app.py

```diff
@@ -202,35 +221,38 @@ def index(category=[]):
     groups = []
     if category:
         if category == 'press-centre':
-            groups = _get_groups_by_slug(slugs='canonical-announcements')
-        else:
-            groups = _get_groups_by_slug(slugs=category)
-            if not groups:
-                groups.append({
-                    "id": -1,
-                    "name": category
-                })
-
-                return flask.render_template(
-                    '404.html'
-                )
+            category = 'canonical-announcements'
+
+        groups = _get_groups_by_slug(slugs=category)
+
+        if not groups:
+            return flask.render_template(
+                '404.html'
+            )
+
+    groups_id = [str(group['id']) for group in groups]
 
     page = flask.request.args.get('page')
     posts, metadata = _get_posts(
-        groups=[str(group['id']) for group in groups],
+        groups=groups_id,
         page=page
     )
```

Comment:
We may want to filter out featured posts from the rest of the posts. Does Insights currently do this?

Reply:
we could remove all the sticky posts from the API call made in the `_get_post` function?

Reply:
Thinking about this, I think this will be fine unless someone requests it.